css/hide_logo.css

```diff
@@ -5,3 +5,6 @@
 .bodycontainer > table tr:first-of-type > td:nth-of-type(2) > font {
   display: none !important;
 }
+
+/* adding a line to remove the gamail.com header */
+@page { size: auto;  margin: 0mm; }
```

Comment:
Let's be consistent with the styling of existing code:

1. Break after `{` and before `}`
2. Break after `;`
3. Indent each property by 2 spaces